scorecard/scorecard/scorecard.py

```diff
@@ -65,34 +59,20 @@
 def index():
     user_data, unassigned, urgent_issues, updated = get_users()
 
-    random_user = random.choice(users)
+    def functional_shuffle(x):
+        x = list(x)
+        random.shuffle(x)
+        return x
 
+    component_users_random = {c: functional_shuffle(us) for c, us in component_users.items()}
     return render_template('index.html', unassigned=unassigned,
-                           user_data=user_data, urgent_issues=urgent_issues, random_user=random_user, updated=updated)
+                           user_data=user_data, urgent_issues=urgent_issues, component_users=component_users_random, updated=updated)
```

Comment:
copy_shuffle is perhaps more descriptive
```suggestion
    def copy_shuffle(x):
         x = list(x)
         random.shuffle(x)
         return x

 
      component_users_random = {c: copy_shuffle(us) for c, us in component_users.items()}
     return render_template('index.html', unassigned=unassigned,		     return render_template('index.html', unassigned=unassigned,
                           user_data=user_data, urgent_issues=urgent_issues, component_users=component_users_random, updated=updated)
```

Reply:
hmm suggestions don't work for multiple lines.